corehq/apps/user_importer/importer.py

```diff
@@ -808,11 +832,18 @@ def process_new_user(self):
             if cv["profile_name"]:
                 _check_profile(cv["profile_name"], self.domain_info.profiles_by_name)
                 profile = self.domain_info.profiles_by_name[cv["profile_name"]]
+            tableau_role = cv["tableau_role"] if "tableau_role" in cv else None
```

Comment:
nit: could this just be `tableau_role = cv["tableau_role"] `, or could just directly use it in `create_or_update_web_user_invite`

Reply:
> could this just be tableau_role = cv["tableau_role"]

We can get a `KeyError` if we don't do a check here.

Reply:
isn't cv just `self.column_values` which will always have `tableau_role` since it's added [here](https://github.com/dimagi/commcare-hq/blob/de4a3e2fbefa6e1a0cac6c287db23e301b4d7f7e/corehq/apps/user_importer/importer.py#L710)?

Reply:
yes, you're right. The `KeyError` I'm remembering was before `tableau_role` was added to `self.column_values`

Reply:
changed here https://github.com/dimagi/commcare-hq/pull/34618/commits/d9c959ce32a405026dbca149b0e5a1faeed5af0c